homework/basic_html_layout/ttx.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TTX</title>
+</head>
+
+<body>
+    <style>
+        .box {
+            display: inline-block;
+            width: 800px;
+            height: 85px;
+            position: relative;
+            background-color: grey;
+            position: relative;
```

Comment:
You can remove `position: relative;` as it is already defined above.

Reply:
thanks , fixed.